locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,72 @@
+# Strings for the breach details checklists
+
+## Prompts the user for changes when there is a breach detected of password
+
+# NOTE: { $companyWebsite } is a placeholder for the URL to the website of the company where the breach occurred 
+breach-checklist-pw-header = Go to <a>{$companyWebsite}</a> to change your password and enable two-factor authentication (2FA). 
```

Comment:
Without a hook into the the anchor's attributes, adding the relevant link is difficult.  I've seen different forms of doing this throughout the old repo, but the link pattern I settled on is similar to this:

```
// locales/en/email-strings.ftl
email-2022-hibp-attribution = Breach data provided by <a { $hibp-link-attr }>{ -brand-HIBP }</a>
```

This way, we have a hook to provide the `href` attribute.  Using Handlebars, I did it like this:
```hbs
// views/partials/email_partials/email-footer.hbs
<p>{{{ getString "email-2022-hibp-attribution" hibp-link-attr='href="https://haveibeenpwned.com/"' }}}</p>
```

With our newest codebase using Template Literals it might be similar to 
```js
`<p>${ getMessage('email-2022-hibp-attribution', { hibp-link-attr: 'href="https://haveibeenpwned.com/"' }) }</p>`
```

Thinking about it further, a shorter naming convention might be better for the new repo:
Instead of ` { $hibp-link-attr }`, we could use an identifier prefix with an "href" sufix, i.e. ` { $hibp-href }`.

cc @flodolo

Reply:
For us, it would be better if we could have just the anchor without any attribute exposed. That way, there's no risk of the variable being altered, and you don't need to retranslate everything if you decide to add other attributes (id, class, etc.). In projects where we used the variable approach, the latter seems to happen a bit too often :-(

Reply:
@mansaj 
Per @flodolo, we can leave this as is, and in a future release add a util function to parse anchor tags adding `href` and any other attributes.  I'll make a note.

Reply:
Sounds good